app/src/main/java/com/github/se/orator/model/symblAi/SpeakingRepositoryRecord.kt

```diff
@@ -56,4 +65,39 @@ class SpeakingRepositoryRecord(private val context: Context) : SpeakingRepositor
   override fun resetRecorder() {
     _analysisState.value = SpeakingRepository.AnalysisState.IDLE
   }
+
+  override fun getTranscript(
+      audioFile: File,
+      onSuccess: (AnalysisData) -> Unit,
+      onFailure: (SpeakingError) -> Unit
+  ) {
+    //        setupAnalysisResultsUsage(onSuccess, onFailure)
```

Comment:
If you're certain that this code won't be used, it's better to delete it rather than commenting it out. It will make the code much cleaner.

Reply:
I'm not the one who kept this comment but it's taken care of now!